infinex-multichain/staging/infinex-multichain-blast.toml

```diff
@@ -0,0 +1,99 @@
+name = "infinex-core-test"
+version = "3.0.0"
+preset = "core"
+description = "Infinex Core Deployments"
+#  pnpm cannon build infinex-multichain/staging/infinex-multichain-blast.toml --chain-id 81457 --dry-run --upgrade-from
+#  pnpm cannon publish infinex-core-test:latest@core --chain-id 81457 --exclude-cloned
```

Comment:
one note is that latest should be changed to the actual version. I think when you publish the latest tag, it doesn't do it correctly.

Reply:
noted